packages/socket-mode/package.json

```diff
@@ -18,8 +18,8 @@
     "state",
     "connection"
   ],
-  "main": "dist/index.js",
-  "types": "./dist/index.d.ts",
+  "main": "dist/src/index.js",
+  "types": "./dist/src/index.d.ts",
```

Comment:
These changes are needed to enable the `resolveJsonModule` option in the `tsconfig.json`, as setting that option changes the output of the TypeScript compiler.